public/400.html

```diff
@@ -1,31 +1,19 @@
 <!DOCTYPE html>
-<!--[if lt IE 9]><html class="lte-ie8" lang="en"><![endif]-->
-<!--[if gt IE 8]><!--><html lang="en"><!--<![endif]-->
+<html lang="en">
     <head>
         <meta charset="utf-8" />
-        <title>This page cannot be found - GOV.UK Contact</title>
+        <title>Error - 400 - Bad request - Track-a-query</title>
```

Comment:
Im interested is there an accessibility reason to change the copy here - is it always a 400 that would cause this pare to load for instance?

Reply:
Yes so it should always load the correct page based on the error that Ruby thinks it is. The title has to reflect that for Google Analytics so we know what happened. The second part is just to be consistent with other pages.